source/reference/modules/deployer/index.rst

```diff
@@ -355,6 +354,122 @@ the steps described in :ref:`debugging-deployer-issues`
   Changing or deleting a processed commit file could cause unchanged files to be indexed again and
   it should be done as a last resort in case of errors.
 
+^^^^^^^^^^^^^^^^^
+Document Indexing
+^^^^^^^^^^^^^^^^^
+Crafter Deployer indexes documents via the following configured items:
+
+- A full text index of any document that has a mimetype that matches the configured list of mimetypes.
+- Indexing of any remote document that matches the configured list of remote documents pattern
+- Indexing of jacketed documents with anything that matches the configured pattern.
+
+Note that indexing of documents in authoring and indexing of documents in delivery each have their own configuration.
```

Comment:
Indexing is done differently in the authoring environment vs the delivery environment.

Authoring indexing is done to help content authors do their work and is controlled by CrafterCMS. The authoring search index is tuned to help authors and is not used by the project/site for delivery concerns.

On the other hand, delivery indexing is done to enable search and search-based features for the delivery project/site. This is configurable per project/site, and the index is tuned to help end-users use the project/site.

Reply:
Let's drop references to authoring; so far, no one other than internal developers cares about this. We may reference this in authoring plugin development. @alhambrav create a ticket for documenting search mechanics for plugin development to be addressed in the future.